sass/directives/_card_blocks.scss

```diff
@@ -0,0 +1,66 @@
+@mixin card-block($include_strip: true) {
+  @include card--tile;
+  @include transform(scale(1)); // TODO mixin that includes transition and transform
+  @include transition(transform 0.4s ease-in);
+
+  padding-top: calc(#{$base-spacing-medium} + #{$small-strip-height});
+  color: $body-text-color;
+  text-align: center;
+  margin: $base-spacing-medium / 2;
+
+  h3 {
+    text-transform: uppercase;
+    font-weight: $font-weight-bold;
+    font-size: $small-font-size;
+    margin-top: $base-spacing-small;
+  }
+
+  &:before {
+    @if $include_strip {
+      @include bs__small-strip;
+    }
+  }
+
+  @media only screen and (min-width: $small-screen-max) { // TODO turning off the fanciness on mobile but should there still be some sort of state for usability?
+    &:hover {
+      @include transform(scale(1.06));
+      @include transition(transform 550ms ease-out);
+
+      &:before {
+        background-color: $anchor-blue;
+      }
+    }
+  }
+}
+
+@mixin card-block__logo {
```

Comment:
I don't think this makes sense as a mixin by itself. the name of the mixin doesn't imply what it does, and it only does one thing which could be handled as a variable. what might be better is 

```scss
@mixin card-block__logo {
  img {
    height: 80px; // this could be a parameter on this mixin, and/or a variable used within the mixin (i'm leaning towards variable, since we might not have cards with varying logo sizes anytime soon, so we can just set a variable for a standard size)
    margin-bottom: $base-spacing-small; // moving this from line 15 above, since we only need this margin if there's a logo present
  }
}
```
this would get included when using the `card--tile` mixin when there's a logo nested in that block, i.e.

```scss
.selector {
  @include card--tile;
  @include card-block__logo; // or rename to card--tile__logo or card__logo
}
```